compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -594,8 +595,10 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             Copy(place) | Move(place) => self.eval_place_to_op(place, layout)?,
 
             Constant(ref constant) => {
+                debug!(?constant);
                 let val =
                     self.subst_from_current_frame_and_normalize_erasing_regions(constant.literal)?;
+                debug!(?val);
```

Comment:
I'd prefer if we wouldn't sprinkle `debug` all over the codebase. That makes it super hard to debug the things I want to look into because all information drowns in noise.

At least this should be made `trace` -- we only want to see this when we are at maximal verbosity level.